topgun/k8s/worker_lifecycle_test.go

```diff
@@ -104,9 +104,12 @@ var _ = Describe("Worker lifecycle", func() {
 					Should(HaveLen(0))
 
 				By("seeing that the build succeeded")
-				startSession := fly.Start("builds", "-j", "some-pipeline/simple-job")
-				<-startSession.Exited
-				Expect(startSession.Out).To(gbytes.Say("succeeded"))
```

Comment:
I think you could also run `fly watch -j some-pipeline/simple-job` and wait for it to exit.

Reply:
oh, good point!